solutions/swb-reference/integration-tests/tests/isolated/projectEnvTypeConfigs/list.test.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('list environment type configs associated to project', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+  const envTypeId = setup.getSettings().get('envTypeId');
+  const projectId = setup.getSettings().get('projectId');
+  const nonExistentProjectId = '12345678-1234-1234-1234-123456789012';
+  const nonExistentEnvTypeId = 'et-prod-1234567890123,pa-1234567890123';
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  test('list projectEnvTypeConfigs excecutes successfully', async () => {
+    const { data: response } = await adminSession.resources.projects
+      .project(projectId)
+      .environmentTypes()
+      .environmentType(envTypeId)
+      .configurations()
+      .get({});
+    expect(Array.isArray(response.data)).toBe(true);
```

Comment:
is there any validation to do on the data?

Reply:
as this is isolated we cannot be certain there will be data, maybe we can add a list on the multiStep and we would be able to validate data

Reply:
Added data check for list on multiStep integ test